release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.3.15.md

```diff
@@ -0,0 +1,5 @@
+## Version 1.3.15.0 2024-03-01
+Compatible with OpenSearch Dashboards 1.3.15
+
+### Maintenance
```

Comment:
We typically include the version bump PR in the maintenance section, and the release notes PR in a `### Documentation` section.

Reply:
since version bump happened as part of same PR as CVEs put it under maintenance